utils/common/src/funTest/kotlin/SafeDeleteRecursivelyFunTest.kt

```diff
@@ -20,7 +20,6 @@
 package org.ossreviewtoolkit.utils.common
```

Comment:
commit-msg: Please provide a short description of the issue and why / how it fixes it.

Reply:
Did this comment go to the wrong commit? This commit is just moving a test.

Reply:
Note that the actual fix is in the next commit, but I'll add some words here as well.